.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,17 @@
+{
+    "configurations": [
```

Comment:
remove .vscode directory and add to gitignore.

Reply:
I removed .vscode directory @Anupam-dagar